gapis/resolve/state_tree.go

```diff
@@ -32,6 +32,15 @@ import (
 	"github.com/google/gapid/gapis/service/path"
 )
 
+const (
+	// If an array or slice has more than this many elements, it is transformed
+	// into multiple tree nodes, each with at most this many children.
+	// The threshold is applied twice, so if the array has more elements than this
+	// number *cubed*, the array root node will contain more than this many child
```

Comment:
> the array root node will contain more than this many child
> nodes (at 2000, that's 8x10^9 elements).

I think I understand what you're saying (that there's no point dealing with arrayGroupThreshold³ ?), but this is rather confusing.

Do you think the inline slice logic is worthwhile? I fear this is rather complex for little gain at the moment - the memory view is probably always going to be a more convenient and powerful visualization. I rather regret adding slice support in here.

If you do think this is worthwhile, how do you feel about putting this variable in the path like you've done for the command tree?